client/signup/config/flows-pure.js

```diff
@@ -437,6 +437,14 @@ export function generateFlows( {
 		showRecaptcha: true,
 	};
 
+	flows[ 'with-design-picker' ] = {
```

Comment:
Note to self: this is how a new flow is defined with the steps that the user will go through